src/components/home/HomeTodaysHot.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Product from 'components/common/Product';
+import { getTodayItemData } from 'libs/api';
+import { TodayItemData } from 'types/types';
+
+function HomeTodaysHot() {
+  const [todayItems, setTodayItems] = useState([] as any);
+  const todaysHotItems = todayItems.slice(0, 3);
+  const {
+    productId,
+    productName,
+    productImage,
+    discount,
+    originalPrice,
+    discountedPrice,
+    reviewCount,
+  }: TodayItemData['data']['todayProductList'] = todayItems;
+
+  const getTodayItems = async () => {
+    const data = await getTodayItemData();
+    setTodayItems(data);
+  };
+
+  useEffect(() => {
+    getTodayItems();
+  }, []);
+
+  return (
+    <>
+      <StDivider />
+      <StTodayItemTitle>
+        <h1>TODAYS HOT</h1>
+        <h2>오늘 가장 잘 나간 상품들을 모아봤어요!</h2>
+      </StTodayItemTitle>
+
+      {todaysHotItems.map((todaysHotItem: any): any => (
+        <Product
+          key={todaysHotItem.productId}
+          productId={todaysHotItem.productId}
+          productName={todaysHotItem.productName}
+          productImage={todaysHotItem.productImage}
+          discount={todaysHotItem.discount}
+          originalPrice={todaysHotItem.originalPrice}
+          discountedPrice={todaysHotItem.discountedPrice}
+          reviewCount={todaysHotItem.reviewCount}
+        />
+      ))}
```

Comment:
```suggestion
      {todaysHotItems.map(({productId, productName, productImage, discount, originalPrice, discountedPrice, reviewCount}) => (
        <Product
          key={productId}
          productId={productId}
          productName={productName}
          productImage={productImage}
          discount={discount}
          originalPrice={originalPrice}
          discountedPrice={discountedPrice}
          reviewCount={reviewCount}
        />
      ))}
```
todaysHotItem. 부분을 줄이고 싶으면 이렇게 작성할 수도 있어용!